dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -130,7 +130,7 @@
 		<UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == '' And '$(_BundlerDebug)' != 'true'">true</UseSystemResourceKeys>
 		<UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == '' And '$(_BundlerDebug)' == 'true'">false</UseSystemResourceKeys>
 		<UseNativeHttpHandler Condition="'$(_PlatformName)' != 'macOS' And '$(UseNativeHttpHandler)' == ''">true</UseNativeHttpHandler>
-		<AutoreleasePoolSupport Condition="'$(AutoreleasePoolSupport)' == ''">true</AutoreleasePoolSupport>
+		<!-- AutoreleasePoolSupport needs to be set earlier, so that illink doesn't override it - https://github.com/dotnet/runtime/pull/86753 - so it's set in Xamarin.Shared.Sdk.props -->
 		<_AggressiveAttributeTrimming Condition="'$(_AggressiveAttributeTrimming)' == ''">true</_AggressiveAttributeTrimming>		
```

Comment:
This may need to be revisited. I assume that if `_AggressiveAttributeTrimming` is set to `true` that ILLink will remove some of the dependency attributes that ILCompiler gets on the input, and thus it can remove user code.

(unless there's a logic which hides this from ILLink and I missed it)

Reply:
Filed to look at it after this PR is in: https://github.com/xamarin/xamarin-macios/issues/18479